mw-webapp/src/styles/_variables.scss

```
@@ -114,3 +110,8 @@ $disableOpacity: 0.7;
     transition: 0.3s;
   }
 }
+
+// animation
+@mixin animation($name, $duration: 1s, $timing-function: ease, $direction: forwards) {
+  animation: $name $duration $timing-function $direction;
+}
```

let's use consistent approach - few different animation for different purposes

```suggestion
@mixin animation( ) {
  animation: defaultAnimationSpecifyMyNameIfPossible 1s ease forwards;
}
```

---

fixed